src/sentry/static/sentry/app/views/alerts/rules/details/relatedIssues.tsx

```diff
@@ -66,10 +65,10 @@ class RelatedIssues extends React.Component<Props> {
             orgId={organization.slug}
             endpointPath={path}
             queryParams={queryParams}
-            query=""
+            query={`start=${start}&end=${end}&groupStatsPeriod=auto`}
```

Comment:
just curious whats the difference between queryParams and query here? seems like they have a lot of the same info

Reply:
Honestly they should be reduced into 1, but as of now we use `queryParams` for issues requests and query to populate the global selection (as well as some parameters for a few requests like `groupStatsPeriod`, not a global selection but still works the same way)